react-css-modules/index.d.ts

```diff
@@ -1,8 +1,9 @@
 // Type definitions for react-css-modules 3.7.9
 // Project: https://github.com/gajus/react-css-modules
-// Definitions by: Kostya Esmukov <https://github.com/KostyaEsmukov>
+// Definitions by: Kostya Esmukov <https://github.com/KostyaEsmukov>, Tadas Dailyda <https://github.com/skirsdeda>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+import * as React from 'react';
 
 declare module 'react-css-modules' {
```

Comment:
The `declare module 'react-css-modules'` one. Just move declarations inside to the upper level. It should work.